specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/devices.json

```diff
@@ -0,0 +1,746 @@
+          }
+        },
+        "mergedToDeviceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The device Id that this device was merged into"
+        },
+        "onboardingStatus": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Onboarded",
+            "NotOnboarded",
+            "NotSupported",
+            "InsufficientInfo"
+          ],
+          "x-ms-enum": {
+            "name": "OnboardingStatus",
+            "modelAsString": true
+          },
+          "description": "Device onboarding status."
+        },
+        "name": {
+          "type": "string",
+          "description": "Device name"
+        },
+        "deviceCategoryId": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "description": "Device category id"
+        },
+        "deviceTypeId": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "description": "Device type id"
+        },
+        "deviceSubTypeId": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "description": "Device sub type id"
+        },
+        "deviceCategoryDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Device category display name"
+        },
+        "deviceTypeDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Device type display name"
+        },
+        "deviceSubTypeDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Device sub type display name"
+        },
+        "profilingConfidence": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Confidence of the device profile"
+        },
+        "riskScore": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "risk score of the device."
+        },
+        "criticality": {
+          "type": "string",
+          "enum": [
+            "Low",
+            "Normal",
+            "High"
+          ],
+          "x-ms-enum": {
+            "name": "Criticality",
+            "modelAsString": true
+          },
+          "description": "Device criticality."
+        },
+        "tags": {
+          "type": "array",
+          "description": "Device tags",
+          "items": {
+            "type": "string"
+          }
+        },
+        "firstSeen": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "First time the device was seen."
+        },
+        "lastSeen": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Last time the device was seen."
+        },
+        "lastUpdated": {
```

Comment:
is this going to differ from what the user sees in systemData?  Why?

Reply:
lastUpdated here describes the last time the system that discovers the device updated the device data. In the near future we will add the option for the user to edit some of the fields and the field "lastUpdated" will not be affected by those updates (only the time in the system data).